src/ducks/apps/components/InstallModal.jsx

```diff
@@ -9,10 +9,19 @@ import { hasPendingUpdate } from 'ducks/apps/appStatus'
 export class InstallModal extends Component {
   constructor(props) {
     super(props)
-    const { app, onAlreadyInstalled } = this.props
-
     this.state = { activeTrap: true }
+  }
+
+  componentDidMount = () => {
+    this.handleInstalledStatus()
+  }
+
+  componentDidUpdate = () => {
```

Comment:
When does this modal update ?

Reply:
After an install, the modal can be udpdated and we was checking the app status only in the constructor